apps/src/lib/tools/jsinterpreter/JSInterpreter.js

```diff
@@ -790,7 +835,8 @@ export default class JSInterpreter {
         }
       } else {
         if (this.executionError) {
-          this.handleError(inUserCode ? userCodeRow + 1 : undefined);
+          const row = inUserCode || !!libraryName ? userCodeRow + 1 : undefined;
```

Comment:
Is row a boolean, number, or undefined depending on the values of these variables? This is kind of a complicated expression so a comment may be helpful.